app/services/needs_assignee_notifier.rb

```diff
@@ -1,8 +1,9 @@
 class NeedsAssigneeNotifier
   def self.notify_new_assignee(need)
-    return unless need.saved_change_to_user_id? || need.saved_change_to_role_id?
+    return unless need.saved_change_to_user_id? || need.saved_change_to_role_id? || need.send_email
```

Comment:
I think this should maybe be `(x || y) && z` otherwise this will send an email if the send_email flag is true, even if one of the changes hasn't happened.

Reply:
I changed it